cmd/internal/server/core.go

```diff
@@ -129,6 +146,14 @@ func (s *Server) shutdown() {
 	// Close any open journals on shutdown to ensure remaining entries are flushed to disk.
 	activity.CloseAll()
 
+	// Flush any outstanding analytics hits
+	ticker := time.NewTicker(analytics.FlushDuration)
+	defer ticker.Stop()
+	go func() {
+		s.analyticsClient.Flush()
+	}()
+	<-ticker.C
```

Comment:
This waits a full 250ms, right? We should stop the ticker if the goroutine completes before then.